src/base/bittorrent/sessionimpl.cpp

```diff
@@ -5200,6 +5211,45 @@ void SessionImpl::loadCategories()
     }
 }
 
+void SessionImpl::loadTorrentParamRules()
+{
+    const Path path = specialFolderLocation(SpecialFolder::Config) / TORRENT_PARAM_RULES_FILE_NAME;
+    const QString pathStr = path.toString();
+    if (!path.exists())
+    {
+        LogMsg(tr("TorrentParamRules not found at \"%1\"").arg(pathStr), Log::INFO);
```

Comment:
There should be user-friendly name of this subsystem/feature defined that would be used in UI and other user-visible strings instead of internals like class name etc.

Reply:
I switched it to just "torrent rules" for now. Let me know if you have a better name in mind.

Reply:
Maybe "Auto torrent configurator" or "Auto torrent customizer"?

Reply:
I switched log messages to "auto torrent customizer" and renamed the config file to `auto_torrent_customizer_rules.json`